src/test/java/org/broadinstitute/hellbender/tools/copynumber/FilterIntervalsIntegrationTest.java

```diff
@@ -262,20 +267,25 @@ public void testCountBasedFilters(final File intervalsFile,
         }
 
         return new Object[][]{
-                //intervals file, annotated-intervals file, min/max GC content, mix/max mappability, min/max seg-dupe content,
+                //intervals file, array of strings for excluded intervals, annotated-intervals file, min/max GC content, mix/max mappability, min/max seg-dupe content,
                 //count files, lowCountFilterCountThreshold, lowCountFilterPercentageOfSamples,
                 //extremeCountFilterMinimumPercentile, extremeCountFilterMaximumPercentile, extremeCountFilterPercentageOfSamples,
                 //expected array of indices of retained intervals
-                {intervalsFile, annotatedIntervalsFile, 0.1, 0.9, 0.1, 0.9, 0.1, 0.9,
+                {intervalsFile, Collections.emptyList(), annotatedIntervalsFile, 0.1, 0.9, 0.1, 0.9, 0.1, 0.9,
                         countFiles, lowCountFilterCountThreshold, percentageOfSamples, 1., 99., percentageOfSamples,
                         IntStream.range(numIntervalsBelowCountThreshold + 1, numIntervalsBelowCountThreshold + 99).boxed().collect(Collectors.toList())},
-                {intervalsFile, annotatedIntervalsFile, 0.1, 0.9, 0.1, 0.9, 0.1, 0.9,
+                {intervalsFile, Collections.emptyList(), annotatedIntervalsFile, 0.1, 0.9, 0.1, 0.9, 0.1, 0.9,
```

Comment:
It would be nice to have a test case where the annotatedIntervalsFile and countFiles is missing a few entries, up to you!